docs/changelog.md

```diff
@@ -255,6 +255,17 @@ what we publish.
   # pw_gecos='System Administrator', pw_dir='/var/root', pw_shell='/bin/zsh')
   ```
 
+- Added `SIMD.count` method for `Scalar` rhs.
+  ([PR #3179](https://github.com/modularml/mojo/pull/3179) by [@rd4com](https://github.com/rd4com))
+
+  Example usage:
+
+  ```mojo
+  var B = SIMD[DType.int64, 4](1)
```

Comment:
It will look better IMHO with some more meaningful name than just "B". It is also a instance like, so snake_case here

Reply:
Just write `v` or `a` here for conformance of the naming guidelines, I guess.

Reply:
@gryznar , @soraros , fixed with new commit ([2adda8f](https://github.com/modularml/mojo/pull/3179/commits/2adda8f34908e51f32d58941093e66fc753d29ae))